src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -38,28 +42,35 @@ export const getInstallations = async (client: GitHubAPI, subscriptions: Subscri
 	};
 };
 
-const getInstallation = async (client: GitHubAPI, subscription: Subscription, log?: Logger): Promise<AppInstallation> => {
-	const id = subscription.gitHubInstallationId;
+const getInstallation = async (client: GitHubAPI, subscription: Subscription, log: Logger): Promise<AppInstallation> => {
+
+	const { jiraHost } = subscription;
+	const useNewGitHubClient = await booleanFlag(BooleanFlags.USE_NEW_GITHUB_CLIENT_FOR_GET_INSTALLATION, false, jiraHost) ;
+	const { gitHubInstallationId } = subscription;
+	const gitHubAppClient = new GitHubAppClient(getCloudInstallationId(gitHubInstallationId), log);
+
 	try {
-		const response = await client.apps.getInstallation({ installation_id: id });
+		const response = useNewGitHubClient ?
+			await gitHubAppClient.getInstallation(gitHubInstallationId) :
+			await client.apps.getInstallation({ installation_id: gitHubInstallationId });
 
 		return {
 			...response.data,
 			syncStatus: mapSyncStatus(subscription.syncStatus),
 			syncWarning: subscription.syncWarning,
 			totalNumberOfRepos: await RepoSyncState.countFromSubscription(subscription),
 			numberOfSyncedRepos: await RepoSyncState.countSyncedReposFromSubscription(subscription),
-			jiraHost: subscription.jiraHost
+			jiraHost
 		};
 
 	} catch (err) {
 		log?.error(
-			{ installationId: id, error: err, uninstalled: err.code === 404 },
+			{ installationId: gitHubInstallationId, error: err, uninstalled: err.code === 404 },
 			"Failed connection"
 		);
 		statsd.increment(metricError.failedConnection);
 
-		return Promise.reject({ error: err, id, deleted: err.code === 404 });
+		return Promise.reject({ error: err, gitHubInstallationId, deleted: err.code === 404 });
```

Comment:
Is this right? I don't see the change to the `FailedAppInstallation` interface to use `gitHubInstallationId` instead of `id`

Reply:
Turns out. You are correct. Will update to id: gitHubInstallationId